index.html

```diff
@@ -248,28 +248,30 @@ <h2 class="contact-head">
         <input
           type="text"
           name="name"
+          aria-label="Name"
           maxlength="30"
           placeholder="Full name"
           required
         />
-        <input type="email" name="email" placeholder="Email address" required />
+        <input type="email" name="email" aria-label="Email" placeholder="Email address" required />
         <textarea
           class="message-box"
           name="message"
+          aria-label="Message"
           maxlength="500"
           placeholder="Enter your message..."
           required
         ></textarea>
-        <input id="button-submit" type="submit" value="Get in touch" />
+        <input id="button-submit" type="submit" aria-label="Submit" value="Get in touch" />
       </form>
     </section>
     <footer>
       <ul class="social-icons">
-        <li><img src="./img/social-icons/dark-icons/github.png" alt="github"></li>
-        <li><img src="./img/social-icons/dark-icons/Linkedin.png" alt="linkedin"></li>
+        <li><img src="./img/social-icons/dark-icons/github.png" alt="github logo"></li>
+        <li><img src="./img/social-icons/dark-icons/Linkedin.png" alt="linkedin logo"></li>
         <li><img src="./img/social-icons/dark-icons/Vectordark-icons-2.png" alt=""></li>
```

Comment:
- [ ] Please kindly add a missing alt text

Reply:
Going to add it. Thanks